### ISSUE TITLE
Add more info to funding fee exemption on funding fee and closing costs page

### ISSUE THREAD

**Issue by RLHecht** [department-of-veterans-affairs/va.gov-team#48028]
## Issue Description
Stakeholder Raleigh Contreras asked us to add clarifying information about paying funding fees when a service member has a proposed or memorandum rating. This work should be done on the [VA funding fee and loan closing costs](https://www.va.gov/housing-assistance/home-loans/funding-fee-and-closing-costs/) page. 

The additional info he wants added is bolded in the fourth bullet below:

H3: Will I have to pay the VA funding fee?

If you’re using a VA home loan to buy, build, improve, or repair a home or to refinance a mortgage, you’ll need to pay the VA funding fee unless you meet certain requirements.

**You won’t have to pay a VA funding fee if any of the below descriptions is true. You’re:**

- Receiving VA compensation for a service-connected disability, **or**
- Eligible to receive VA compensation for a service-connected disability, but you’re receiving retirement or active-duty pay instead, **or** 
- The surviving spouse of a Veteran who died in service or from a service-connected disability, or who was totally disabled, **and** you're receiving Dependency and Indemnity Compensation (DIC), **or**
- A service member with a proposed or memorandum rating, before the loan closing date, saying you're eligible to get compensation because of a pre-discharge claim (**If a proposed or memorandum rating is not obtained and a closing takes place, the funding fee exemption does not apply, and the Service member will not be entitled to a refund)**, **or**
- A service member on active duty who before or on the loan closing date provides evidence of having received the Purple Heart

We need to edit this content to make it plain language and to adhere to our style guide. We also need to rework the bullet list setup so that it follows the pattern we've been recently doing ("You're" is at the start of each bullet, no sighted language) and fix the bullet with a bold "and" in the middle of it. There are most likely other edits to the page we want to make to have it more align with our recent patterns, but we will create a separate ticket for that once this work is complete.

## Tasks
- [x] Edit content and add to Drupal page
- [x] Review (Laura)
- [x] Copyedit (Randi)
- [x] Send to stakeholder for approval (Laura)
- [x] Prepub checklist (Megan)
- [x] Publish (Megan)

## Acceptance Criteria
- [x] New funding fee exemption info is on live page

**Comment by aliyahblackmore** [comment]
Updating this ticket cc: @laurwill !
Preview

**I made these changes in Drupal (based on our Thursday 10/13 sync and slack messages) to move this along:**

Changed the sentence before the bulleted list to be aligned with current VA style standards and added **“You're”** to the beginning of each bullet point. Also, as you suggested, the bullet point that previously had “and” in the middle was changed to “You’re receiving Dependency and Indemnity Compensation (DIC) as the surviving spouse of a Veteran, or”

**As discussed, we'll hold off on the other changes until the stakeholder confirms these questions:** 

What is a proposed or memorandum rating? 

If you're a service member who wants to use this benefit while you're still in service you either have to get a Purple Heart or a proposed or memorandum rating before loan closing. But what happens if you're a service member who doesn't get the proposed or memorandum rating before the loan closing but then you get VA disability compensation?

**Comment by laurwill** [comment]
Hi @RLHecht , 
Aliyah and I chatted and we think we came up with a logical solution here, so we can send Raleigh our proposed edits and ask him to check for accuracy. Since we changed a bunch of text, could you copyedit just the "Will I have to pay the fee?" section before we send back to Raleigh?
http://preview-prod.vfs.va.gov/preview?nodeId=748

**Comment by RLHecht** [comment]
This looks good to me @laurwill. You can send to Raleigh!

**Comment by laurwill** [comment]
Thanks Randi! Sent to Raleigh for accuracy review.

**Comment by laurwill** [comment]
Hi @megzehn , I moved the info from the note into the main text of the "refunds" section, as SME requested and we chatted about in standup. If that looks good to you, this is ready for a final copyedit and publishing today!

Noting that we only made changes to the "will i have to pay this fee" section — we have a backlog ticket to look at updates to the page as a whole.

http://preview-prod.vfs.va.gov/preview?nodeId=748

**Comment by megzehn** [comment]
Thanks, @laurwill! 

Confirming that this has been copy edited and published. CC @RLHecht

**Comment by laurwill** [comment]
Validated and notified stakeholder. We can close @RLHecht !

**Comment by RLHecht** [comment]
This is live. Closing ticket.
